docs/compatibility.txt

```diff
@@ -0,0 +1,59 @@
+=============
+Compatibility
+=============
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+JDBC
+====
+
+This JDBC driver follows the JDBC 4.1 standard as it uses PGSQL driver as the
```

Comment:
This is a bit off as we are not using the pgsql driver but  we speak the postgres wire protocol. Other than that it's our own implementation

Reply:
fixed